testing/BUILD.bazel

```diff
@@ -7,16 +8,60 @@ package(default_visibility = ["//visibility:public"])
 kt_android_library(
      name = "testing_main_lib",
```

Comment:
Prefer not setting the default visibility for a package. We should lock down access so that only the things that need to be public are made public.

Reply:
Done!